georocket-server/integration-tests/src/XMLTests.groovy

```diff
@@ -152,4 +152,99 @@ class XMLTests extends StoreTests {
             + "?search=TestBuilding2", null, true)
         assertExportedBuilding(exportedContents, 'ID_146_D')
     }
+
+    /**
+     * Add and delete tags
+     */
+    def testTags() {
+        // ensure that no chunk with the test tags is found
+        assertNotFound("test1")
+        assertNotFound("test2")
+        assertNotFound("test3")
+
+        // append single test tag to the chunk
+        run("curl -sS -X PUT http://${georocketHost}:63020/tags/"
+                + "?search=ID_146_D&tags=test1", null, true)
+        sleep(1000)
```

Comment:
Is this necessary? What if Travis CI is too slow (which it sometimes is) and the operation takes longer than 1000ms?